src/local_command.rs

```diff
@@ -10,6 +10,36 @@ use bevy::prelude::*;
 #[derive(Component)]
 pub struct LocalCommand {
     pub(crate) command: Command,
+    pub(crate) delay: Option<Timer>,
+    pub(crate) state: LocalCommandState,
+}
+
+/// Keep track of the state of the running process.
+///
+/// Ready - Process is ready to be run - pending start time.
+/// Running - Process is running.
+/// Error - Process errored out. Allows for retry logic to kick in. Otherwise state moves to Done
+/// Done - Process has completed. Final state, allows for cleanup logic.
```

Comment:
Nitpick: I'd suggest to move these doc comments directly above the corresponding enum variants, this will work better with the docs.rs output

Reply:
Same for `LocalCommandDone`

Reply:
Got it!